### PR TITLE
Implement hello_world.cpp

### PR DESCRIPTION
**Pull request by spostman**
review: @spost-avatar

### CODE REVIEW

File: hello_world.cpp [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#include <stdio.h>
+
+int main(int argc, char** argv) {
+  printf("Hello World!");
```
Comment (spost-avatar) [inline hello_world.cpp]: Please leave the line feed character \n

File: hello_world.cpp [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#include <stdio.h>
```
Comment (spost-avatar) [inline hello_world.cpp:2]: At the beginning the file, you should explain what this program is supposed to do.

**Review by spost-avatar — COMMENTED** [review]
Please address my comments

File: hello_world.cpp (hunk 2 above)
Comment (spostman) [reply hello_world.cpp:2]: Done

File: hello_world.cpp (hunk 1 above)
Comment (spostman) [reply hello_world.cpp]: Done.

**Review by spost-avatar — APPROVED** [review]
lgtm

**Pull request merged**